### DISCUSSION TITLE
Collecting reports from multiple tests into a single folder

### DISCUSSION THREAD

**Discussion by michaelplavnik** [VIP-Logic/NBomber#734] [General]
From usability perspective, I need to collect reports for several executions of NBomber into a single folder. 
It seems by default report folder is cleaned.

Is it possible to control this behavior?  Might be you can add a parameter to WithReportFolder(path, autoClean: false)

**Comment by AntyaDev** [comment]
Hi @michaelplavnik ,
Yes, it is possible for sure.

I recall this task: https://github.com/PragmaticFlow/NBomber/issues/643
Maybe it can be helpful in your scenario to keep everything in one HTML report

@michaelplavnik Can you please register a task in our backlog?
https://github.com/PragmaticFlow/NBomber/issues
